src/lib/REST/Output/ValueObjectVisitor/ContentTree/NodeExtendedInfo.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\REST\Output\ValueObjectVisitor\ContentTree;
+
+use Ibexa\Contracts\Rest\Output\Generator;
+use Ibexa\Contracts\Rest\Output\ValueObjectVisitor;
+use Ibexa\Contracts\Rest\Output\Visitor;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @phpstan-import-type TPermissionRestrictions from \Ibexa\AdminUi\REST\Value\ContentTree\NodeExtendedInfo
+ */
+final class NodeExtendedInfo extends ValueObjectVisitor
+{
+    /**
+     * @param \Ibexa\AdminUi\REST\Value\ContentTree\NodeExtendedInfo $data
+     */
+    public function visit(Visitor $visitor, Generator $generator, $data): void
+    {
+        $generator->startObjectElement('ContentTreeNodeExtendedInfo');
+        $visitor->setHeader('Content-Type', $generator->getMediaType('ContentTreeNodeExtendedInfo'));
```

Comment:
This also could be a `public const MEDIA_TYPE`. Semantically it's not the same as object element one, so not suggesting to use the same, but maybe I'm overthinking this ;)